### ISSUE TITLE
[Enhancement] default `params` type suggestion for `WebSocket`

### ISSUE THREAD

**Issue by bogeychan** [elysiajs/elysia#312]
### Description

The following code reports a typescript type-error: (_same with all other `WebSocket`-hooks_)

```ts
new Elysia().ws('/:id', {
  open(ws) {
    ws.data.params.id;
  }
});
```

```txt
Property 'id' does not exist on type 'never'.ts(2339)
```

Unlike a regular HTTP-Handler:

```ts
new Elysia().get('/:id', (ctx) => {
  ctx.params.id; // (property) id: string
});
```

### Environment

- `bun@1.0.11`
- `elysia@0.7.26`

### Additional Informations

overwriting the `params` type works:

```ts
new Elysia().ws('/:id', {
  params: t.Object({
    id: t.String()
  }),
  message(ws) {
    ws.data.params.id; // (property) id: string
  }
});
```

**Comment by SaltyAom** [comment]
Fix with eabe264, available on 0.7.27.

Feels free to reopen if the issue still persists.

**Comment by bogeychan** [comment]
Works!

but if i override it, since `0.7.27`:

```ts
new Elysia().ws('/:id', {
  params: t.Object({
    id: t.Numeric()
  }),
  open(ws) {
    console.log(ws.data.params.id); // (property) id: never
    // (property) params: { id: number; } & Record<"id", string>
  }
});
```
